e2e/build_uki_test.go

```diff
@@ -26,9 +26,13 @@ var _ = Describe("build-uki", func() {
 		currentDir, err := os.Getwd()
 		Expect(err).ToNot(HaveOccurred())
 		keysDir = filepath.Join(currentDir, "assets", "keys")
+		Expect(os.MkdirAll(keysDir, 0755)).ToNot(HaveOccurred())
 
 		enki = NewEnki("enki-image", resultDir, keysDir)
 		image = fmt.Sprintf("quay.io/kairos/fedora:38-core-amd64-generic-%s", kairosVersion)
+		// Generate keys
+		run, err := enki.Run("genkeys", "--output", keysDir, "TEST")
+		Expect(err).ToNot(HaveOccurred(), run)
```

Comment:
is it worth checking if the keys are present, or a count of the generated files in there?

Reply:
The needed check is already during the enki run so I would think is useless, plus it will fail if it can't generate the keys anyway